cmd/containerd/command/config.go

```diff
@@ -120,7 +120,7 @@ var configCommand = cli.Command{
 
 func platformAgnosticDefaultConfig() *srvconfig.Config {
 	return &srvconfig.Config{
-		Version: 2,
+		Version: 1,
```

Comment:
suggest adding a comment here something like 
```
// see: https://github.com/containerd/containerd/blob/5c6ea7fdc1247939edaddb1eba62a94527418687/RELEASES.md#daemon-configuration  
// this version MUST remain set to 1 until either there exists a means to override / configure the default at the containerd cli .. or when 
// version 1 is no longer supported  
```

Reply:
Thanks for the review. 
Added the suggested comment.